projects/packages/forms/src/types/index.ts

```diff
@@ -29,7 +29,7 @@ export interface Integration {
 /**
  * Props for integration cards in the Jetpack Forms dashboard and integrations modal.
  */
-export interface IntegrationCardProps {
+export interface SingleIntegrationCardProps {
```

Comment:
Updated this name since we're now using IntegrationCardProps as the type for the common IntegrationCard component. Since this is updated, then had to update imports in the files in dashboard/integrations above.